CHANGELOG.md

```diff
@@ -6,14 +6,26 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres
 to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [UNRELEASED]
+## [3.2.0][3.2.0]
+
+### Added
+
+* Added field Liability to AdditionalTransactionData.Card for non-3ds card transactions  
 
 ### Removed
 
 * Removed resource `version.properties`. Version is set at generate-sources phase.
 * Removed class `com.unzer.payment.exceptions.PropertiesException`
 * Removed deprecated method `com.unzer.payment.util.SDKInfo::getVersion()`. Please, use `com.unzer.payment.util.SDKInfo.VERSION` instead 
 
+### Deprecated
+
+* Deprecated constructor with args `com.unzer.payment.models.CardTransactionData.CardTransactionData(com.unzer.payment.enums.RecurrenceType)`. Use empty one instead
```

Comment:
Wouldnt it be easier for merchant to use this constructor, though. It is the only value he has to set for card data.